robot-core/src/main/java/org/obolibrary/robot/Template.java

```diff
@@ -559,33 +559,23 @@ private void addTable(List<List<String>> rows) throws Exception {
 
   /** Add the labels from the rows of the template to the QuotedEntityChecker. */
   private void addLabels() {
-    // If there's no label column, we can't add labels
-    if (labelColumn == -1) {
-      return;
-    }
```

Comment:
Removing this will necessitate looping through potentially a lot of rows for no reason (the phrase "for no reason" applies to the "looping", not the "removing"); I am assuming you are removing this because you want to add the "entity --> id" combination to the EntityChecker, right?

Reply:
What your test case here, #1133, and #1156 all have in common is that none of you are using a LABEL column. I had assumed that everyone would want a LABEL and would use the label in Manchester strings, but in these cases an ID and a TYPE are enough.

So we will change the behaviour so that every row ends up in the QuotedEntityChecker, and that fixes these three problems. I *think* it's a safe change.

Reply:
Yes, makes sense! I agree with you, just wanted to make sure we both thought this way!